Dockerfile

```diff
@@ -10,106 +9,183 @@ COPY test /usr/src/app/test
+########################################################
+
+FROM eget-builder AS process-exporter
+
 # renovate: datasource=github-releases depName=ncabatoff/process-exporter
 ARG PROCESS_EXPORTER_TAG=0.7.10
+
+RUN eget ncabatoff/process-exporter \
+	--tag v${PROCESS_EXPORTER_TAG} --asset ".tar.gz" \
+    --file "process-exporter" --to "/usr/local/bin/"
+
+########################################################
+# Openvpn Exporter
+########################################################
+
+FROM eget-builder AS openvpn-exporter
+
 # renovate: datasource=github-releases depName=natrontech/openvpn-exporter
 ARG OPENVPN_EXPORTER_TAG=1.0.2
 
-RUN eget prometheus/node_exporter --tag v${NODE_EXPORTER_TAG} \
-	&& eget ncabatoff/process-exporter --tag v${PROCESS_EXPORTER_TAG} \
-	&& eget natrontech/openvpn-exporter --tag v${OPENVPN_EXPORTER_TAG} \
-	&& mv /usr/local/bin/openvpn-exporter-linux-${TARGETARCH:-amd64} /usr/local/bin/openvpn-exporter
+RUN eget natrontech/openvpn-exporter \
+	--tag v${OPENVPN_EXPORTER_TAG} --asset ".tar.gz" --asset "^sbom" \
+    --file "openvpn-exporter-linux-*" --to "/usr/local/bin/openvpn-exporter"
 
-EXPOSE 80 443 3128
+########################################################
+# Sshproxy
+########################################################
 
-# hadolint ignore=DL3008
-RUN apt-get update && apt-get install -y --no-install-recommends \
-	socat \
-	&& rm -rf /var/lib/apt/lists/*
+FROM eget-builder AS sshproxy
+
+# renovate: datasource=github-releases depName=balena-io/sshproxy
+ARG SSHPROXY_TAG=1.4.1
+
+RUN eget balena-io/sshproxy --tag v${SSHPROXY_TAG} \
+	--asset ".tar.gz" --file "*" --to "/usr/local/bin/"
+
+########################################################
+# Libnss-openvpn
+########################################################
+
+FROM plugin-builder AS libnss-openvpn
+
+ENV LIBNSS_OPENVPN_VERSION=22feb11322182f6fd79f85cd014b65b6c40b7b47
+
+WORKDIR /usr/src/app/libnss-openvpn
+RUN git clone https://github.com/balena-io-modules/libnss-openvpn.git . \
+	&& git checkout ${LIBNSS_OPENVPN_VERSION} \
+	&& make -j "$(nproc)" \
+	&& make install DESTDIR=/opt
+
+########################################################
+# Main
+########################################################
+
+FROM base AS main
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+COPY --from=node-exporter /usr/local/bin/node_exporter /usr/local/bin/
+COPY --from=process-exporter /usr/local/bin/process-exporter /usr/local/bin/
+COPY --from=openvpn-exporter /usr/local/bin/openvpn-exporter /usr/local/bin/
+COPY --from=sshproxy /usr/local/bin/* /usr/local/bin/
+COPY --from=libnss-openvpn /opt/ /
 
 # https://docs.renovatebot.com/modules/datasource/repology/
 # renovate: datasource=repology depName=debian_12/haproxy versioning=loose
 ARG HAPROXY_VERSION=2.6.12-1+deb12u2
 
+# https://docs.renovatebot.com/modules/datasource/repology/
+# renovate: datasource=repology depName=debian_12/openvpn versioning=loose
+ARG OPENVPN_VERSION=2.6.3-1+deb12u3
```

Comment:
This used to be a floating version in open-balena-base, but honestly we should be pinning it here for such an important dependency.